### PR TITLE
[Filters] Fix view based filters when string input is enabled

### LINKED ISSUE #7006: [Filters] Filters without the "possibleValues" and string values are causing errors
#### Summary
When utilizing filters (https://github.com/nasa/openmct/tree/master/src/plugins/filters) without a `possibleValues` key (to be able to accept string values), an error is thrown here: https://github.com/nasa/openmct/blob/master/src/plugins/telemetryTable/components/table-footer-indicator.vue#L161 this is because `comparator` in this instance is a string.

There is also a case where the filters object for any properties that are set up to be strings (because the `possibleValues` is removed) are coming back to the telemetry providers as empty objects and not strings.

#### Expected vs Current Behavior

We need to handle string values in some manner.

#### Steps to Reproduce

1. set up a telemetry provider with filters (https://github.com/nasa/openmct/blob/master/src/plugins/filters/README.md)
2. remove the `possibleValues` property
3. add the telemetry endpoint you added filters for to a table
4. go into edit mode and scroll down to the filters "Global Filters" section and add in a string value
5. observe the error in console

#### Impact Check List

- [ ] Data loss or misrepresented data?
- [ ] Regression? Did this used to work or has it always been broken?
- [ ] Is there a workaround available?
- [ ] Does this impact a critical component?
- [ ] Is this just a visual bug with no functional impact?
- [ ] Does this block the execution of e2e tests?
- [ ] Does this have an impact on Performance?

### PR DESCRIPTION
**Pull request by jvigliotta**
Closes #7006

### Describe your changes:

Modified the logic for filters that do not have the "possibleValues" key in their metadata which allows string input.

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [ ] Tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [x] Changes appear to address issue?
* [x] Reviewer has tested changes by following the provided instructions?
* [x] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [x] Code style and in-line documentation are appropriate?
* [x] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

File: src/plugins/filters/components/FilterField.vue [hunk 1]
```diff
@@ -125,6 +125,7 @@ export default {
       }
     }
   },
+  emits: ['filterTextValueChanged', 'filterSelected', 'clearFilters', 'filterSingleSelected'],
```
Comment (ozyx) [inline src/plugins/filters/components/FilterField.vue:128]: Nice!

File: src/plugins/telemetryTable/TelemetryTable.js [hunk 2]
```diff
@@ -372,6 +376,9 @@ define([
         this.domainObject.configuration.filters &&
         this.domainObject.configuration.filters[keyString];
 
+      // de-reactify it
+      filters = toRaw(filters);
```
Comment (ozyx) [inline src/plugins/telemetryTable/TelemetryTable.js:380]: We might not want to couple this with Vue, is there a way we could ensure the filters are not wrapped in a proxy before being persisted on the config?

**Review by ozyx — CHANGES_REQUESTED** [review]
Have a question for you plus it looks like one of the e2e is failing?

File: src/plugins/telemetryTable/TelemetryTable.js (hunk 2 above)
Comment (jvigliotta) [reply src/plugins/telemetryTable/TelemetryTable.js:380]: Great point. No reason this should be in here, I'll see if it's even needed. The others in the Vue file may be enough.

**Review by ozyx — APPROVED** [review]

**Pull request merged**